Makefile

```diff
@@ -15,9 +15,17 @@ BUILDAH_BUILD ?= buildah bud
 GO ?= go
 GO_BUILD = $(GO) build
 GO_TEST ?= $(GO) test
-KUTTL ?= kubectl-kuttl
+
```

Comment:
:thinking: I like this approach a lot. Did you happen to notice any performance difference using this method? I'm guessing it'd be minimal, but I was curious if you noticed any change.

Reply:
The first time I run one of these, it has to download and compile the code, but it's very cached after that. I don't notice a speed difference myself.